### ISSUE TITLE
InstructorFeedbackEditPage: Add option to discard changes for session details

### ISSUE THREAD

**Issue by VamsiSangam** [TEAMMATES/teammates#7232]
**Description of feature/enhancement**

We need to add a "Discard changes" option for the feedback session details too. Precisely speaking, when we try to edit a feedback question,

![edit-feedback-3](https://cloud.githubusercontent.com/assets/11662001/25746268/5621ab98-31c0-11e7-94b2-751e22e8ffce.JPG)

we get these options -

![edit-feedback-4](https://cloud.githubusercontent.com/assets/11662001/25746210/039e5948-31c0-11e7-9355-b57742ee067d.JPG)

But when we edit the feedback session details, 

![edit-feedback-1](https://cloud.githubusercontent.com/assets/11662001/25746279/63dccd1c-31c0-11e7-9573-991c7557d144.JPG)

we don't get a "Discard changes" option -

![edit-feedback-2](https://cloud.githubusercontent.com/assets/11662001/25746232/1df79e94-31c0-11e7-8be8-df704a147801.JPG)

(Icons appear because I tinkered with the code)

**Justification**

If the instructor wants to discard changes in the feedback details, he has no option but to reload the page, but by doing so, they will lose changes made to the feedback questions.

**Comment by nikiforosbotis** [comment]
I will start working on it

**Comment by HirdayGupta** [comment]
@nikiforosbotis Still working on this? Or may I take this?

**Comment by nikiforosbotis** [comment]
@HirdayGupta No, you are very welcome to take it. Sorry for not letting you know earlier.

**Comment by sukanta-27** [comment]
@damithc I would like to work on this one.

**Comment by damithc** [comment]
> @damithc I would like to work on this one.

Go ahead @sukanta-27

**Comment by wkurniawan07** [comment]
<img width="1232" alt="Screenshot 2022-02-19 at 1 11 25 AM" src="https://user-images.githubusercontent.com/7261051/154730165-91c5f19e-d350-40e8-b298-737288147658.png">

Feature is already there.
